app/controllers/clock_controller.rb

```diff
@@ -0,0 +1,4 @@
+class ClockController < ApplicationController
+  def now
```

Comment:
how about creating an instance variable here for time and then use it in the view?